TLYShyNavBar/TLYShyNavBarManager.h

```diff
@@ -56,6 +56,7 @@
  * Defaults to YES
  */
 @property (nonatomic, getter = isAlphaFadeEnabled) BOOL alphaFadeEnabled;
+@property (nonatomic, getter = isAlphaFadeEntireNavBarEnabled) BOOL alphaFadeEntireNavBar;
```

Comment:
This would be replaced with a property of `TLYShyNavBarFade`, and the previous boolean would be deprecated.